### ISSUE TITLE
Cannot navigate with tab in settings if option is selected

### ISSUE THREAD

**Issue by derekriemer** [microsoft/vscode#85456]
Issue Type: <b>Bug</b>

Using a screen reader:

1. open settings.
2. focus the treeview. arrow down to editor open it, and arrow to cursor.
3. press tab.

expected tab moves to the settings.

actual: Once a section has been expanded, tab goes nowhere.

VS Code version: Code - Insiders 1.41.0-insider (a747d264d668a947eeacd3492b60df6ce8dee6b4, 2019-11-23T11:09:42.834Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2208)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.85GB (6.44GB free)|
|Process Argv|C:\Users\driem\Google Drive\vscode\nvda.code-workspace|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2019.11.50794
remote-containers|ms-|0.85.0
remote-ssh|ms-|0.47.2
remote-ssh-edit|ms-|0.47.2
remote-ssh-explorer|ms-|0.47.2
remote-wsl|ms-|0.41.0
vscode-remote-extensionpack|ms-|0.17.0
cpptools|ms-|0.26.2-insiders2

</details>

**Comment by roblourens** [comment]
@joaomoreno, this is from this line, https://github.com/Microsoft/vscode/commit/1415112d6d4cbf828b0d57dfd2cf3f22c906f3f3#diff-b952af98b71f6319cdacc49df778ffc8L558, I'm not sure why it's removed, can I add it back?

**Comment by joaomoreno** [comment]
@roblourens Sorry! I thought it was used for theming only... My bad.
